src/docs/asciidoc/coreFeatures.adoc

```diff
@@ -225,8 +225,9 @@ work stealing that can happen in some Schedulers.
 Some operators use a specific Scheduler from `Schedulers` by default (and usually give
 you the option of providing a different one). For instance, calling the factory method
 `Flux.interval(Duration.ofMillis(300))` produces a `Flux<Long>` that ticks every 300ms.
-By default, this is enabled by `Schedulers.parallel()` by default. The following line
-changes the Scheduler to a `Single`:
+By default, this is enabled by `Schedulers.parallel()`. The following line changes the
```

Comment:
replace `. The following line changes the` with `, but you could decide to change the`
(that second sentence is now weird: "the following line...as shown in the following example")